lib/context-manager/context.js

```diff
@@ -19,11 +21,38 @@ module.exports = class Context {
     return this._transaction
   }
 
-  enterSegment({ segment, transaction = this.transaction }) {
+  enterSegment({ segment, transaction = this._transaction }) {
     return new this.constructor(transaction, segment)
   }
 
   enterTransaction(transaction) {
     return new this.constructor(transaction, transaction.trace.root)
   }
+
+  // the following methods are used in opentelemetry bridge
+  // see: https://opentelemetry.io/docs/languages/js/context/
+  getValue(key) {
+    return this._otelCtx.get(key)
+  }
```

Comment:
This comment will get out of sync very easily and quickly. Instead, I think we should document each method:

```suggestion
  /**
   * Required for bridging OTEL data into the agent.
   *
   * @param {string} key Stored entity name to retrieve.
   *
   * @returns {*} The stored value.
   */
  getValue(key) {
    return this._otelCtx.get(key)
  }
```